### PR TITLE
Issue template added

### PR DESCRIPTION
**Pull request by Madhan-SWE**
fixes: #99

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Bug report
+about: Create a report to help us improve EBS CSI Driver
```
Comment (mkumatag) [inline .github/ISSUE_TEMPLATE/bug_report.md:3]: Please correct the text to point to powervs driver

**Review by mkumatag — CHANGES_REQUESTED** [review]
minor comment

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (Madhan-SWE) [reply .github/ISSUE_TEMPLATE/bug_report.md]: fixed

**Review by mkumatag — APPROVED** [review]
/lgtm
/approve

**Pull request merged**